README.md

```diff
@@ -0,0 +1,73 @@
+# SpiceRefactor
+
+This library allows for the position, rotation, velocity and rotational velocity tracking of
+multiple bodies in space, especially in relation to one another. It makes extensive use of NAIF's
+SPICE data for such calculations. 
+
+## Setting up dependencies with conda (RECOMMENDED)
+
+Install conda if you do not already have it.
+```bash
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+bash miniconda.sh -b
+```
+> You can add a `-p <install-prefix>` to choose where to install miniconda. By default, it will install it to `$HOME/miniconda3`.
+
+### Setting up conda for bash
+Copy and paste the following into a terminal running the `bash` shell:
+```bash
+echo -e "\n\n# Adding miniconda3 to PATH" >> $HOME/.bashrc && \
+echo -e "export PATH=$HOME/miniconda3/bin:\$PATH" >> $HOME/.bashrc && \
+source $HOME/.bashrc && \
+which conda
+```
+> *For more information: [bash installation](https://conda.io/docs/user-guide/install/linux.html "Reference to bash conda install")*
+
+### Setting up conda for tcsh
+Copy and paste the following into a terminal running the `tcsh` shell:
+```tcsh
+echo  "\n\n# Setting up miniconda3 for tcsh" >> $HOME/.cshrc && \
+echo  "source $HOME/miniconda3/etc/profile.d/conda.csh > /dev/null" >> $HOME/.cshrc && \
+source $HOME/.cshrc && \
+which conda
+```
+> *For more information: [tcsh installation](https://github.com/ESMValGroup/ESMValTool/issues/301 "Reference to tcsh conda install")*
```

Comment:
Can we link to an Anaconda install guide instead of writing the instructions out here?